.travis.yml

```diff
@@ -26,6 +26,10 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
     - php: 5.6
+      env: SYMFONY_VERSION=3.1.*
+    - php: 5.6
+      env: SYMFONY_VERSION=3.2.*
+    - php: 5.6
```

Comment:
Can't reproduce this locally event with composer version b1156ed376adc39de57f4bb4ed2c345ae8cbeb7e (snapshot-channel). Try adding this: `composer self-update --stable`

Profile output:
```
[468.7MB/73.19s]   - Installing symfony/symfony (dev-master 9f95654)[468.7MB/174.70s]  Cloning 9f95654515 from cache
```